apps/vaporgui/CMakeLists.txt

```diff
@@ -271,8 +272,20 @@ else ()
 	add_executable (vapor ${SRCS} ${UIS})
 endif()
 
-# target_link_libraries (vapor common vdc wasp render params m jpeg dl util GLEW tiff geotiff assimp Qt4::QtCore Qt4::QtGui Qt4::QtOpenGL)
-target_link_libraries (vapor common vdc wasp render params jpeg ${GLEW} tiff geotiff assimp Qt4::QtCore Qt4::QtGui Qt4::QtOpenGL python${PYTHONVERSION})
+if (WIN32)
+	target_link_libraries (
+    		vapor common vdc wasp render params jpeg ${GLEW} 
+    		libtiff geotiff assimp-vc140-mt Qt5::Widgets Qt5::OpenGL Qt5::Core Qt5::Gui Qt5::DBus 
```

Comment:
Instead of having all these additional ifdefs, could you just create a variable ASSIMP_LIB and set it to "assimp-vc140-mt" on windows and "assume" everywhere else? (why does it have to be called "assimp-vc140-mt")

Reply:
Done.  "assimp-vc140-mt" is the name of the library created in VS2014.  I think we should stick to default options when building the Win32 libraries for simplicity.